app/controllers/invitations_controller.rb

```diff
@@ -34,11 +60,15 @@ def accept
 
   private
 
-  def find_invitation
+  def find_active_invitation
     @invitation = Invitation.active.find_by(token: params[:token])
     redirect_to root_path, alert: 'Bad invitation token' unless @invitation
   end
 
+  def find_invitation
+    @invitation = Invitation.find_by!(token: params[:token])
```

Comment:
@shir current_organization is nil at this place because new user not signed in yet